packages/salesforcedx-vscode-apex-replay-debugger/test/vscode-integration/commands/quickLaunch.test.ts

```diff
@@ -361,9 +362,14 @@ describe('Quick launch apex tests', () => {
 
     expect(notificationServiceStub.called).to.equal(true);
     const notificationArgs = notificationServiceStub.getCall(0).args;
-    expect(notificationArgs[0]).to.equal(
-      "Cannot read property 'length' of undefined"
+    // Seems that there are two different error msgs:
+    // On Windows: "Cannot read property 'length' of undefined"
+    // On Mac: "Cannot read properties of undefined (reading 'length')"
+    expect(notificationArgs[0].startsWith('Cannot read propert')).to.equal(
```

Comment:
@gbockus-sf FYI this test passes locally and fails the circle build due to the msg being different.  I tried to make the test work for both, but let me know if you think a different approach would be better here.

Reply:
seems fine.  weird that it's different between mac & windows.